src/plugins/multimedia/multimedia.js

```diff
@@ -680,19 +681,50 @@ $document.on( initializedEvent, selector, function( event ) {
 
 			// finally lets load safely
 			return Modernizr.load( {
-				load: "https://www.youtube.com/iframe_api"
+				load: "https://www.youtube.com/iframe_api",
+
+				//possible solution for multimedia and doaction conflict in corporate network
+				complete: function() {
+					setTimeout( function() {
+						var resources, arrIframesYt, notifText;
```

Comment:
We prefix our variable that contain jQuery object with "$" 
```suggestion
						var resources, arrIframesYt, $notifText;
```

Reply:
Done. Thanks